drivers/windows/windows.go

```diff
@@ -208,6 +211,18 @@ func (d *driver) parseNetworkOptions(id string, genericOptions map[string]string
 				return nil, err
 			}
 			config.VSID = uint(vsid)
+		case EnableOutboundNat:
+			if system.GetOSVersion().Build <= 16236 {
```

Comment:
Can the build number `16236` be defined as a global constant? It seems to be used at a couple of places.